### PR TITLE
Shell those nuts!

### PR DESCRIPTION
**Pull request by RDru**
#### Summary
```SUMMARY: Features "Shelling nuts by recipes"```

#### Purpose of change
Autumn came. Gathered some nuts... 6m to disassemble them 1 by 1?! That's a lot of time and clicking.

#### Describe the solution
New auto-learn recipes for shelling nuts. 5s and 10s (hammer) for each nut.
Removed nut disassembling.

#### Testing
Shelled all.

### CODE REVIEW

File: data/json/recipes/food/unshell.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "type": "recipe",
+    "result": "pistachio_unshelled",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "skills_required": [ "survival", 0 ],
```
Comment (Rail-Runner) [inline data/json/recipes/food/unshell.json]: Don't think it changes anything to specify needed skill level of 0; could just remove it.

Comment (RDru) [reply data/json/recipes/food/unshell.json]: I wanted it to unlock at 0 but train cooking a bit. I just tested and it doesn't unlock at 0.

Comment (RDru) [reply data/json/recipes/food/unshell.json]: Ok, removed, also difficulty. Unlocks at 0 now.

**Comment by ZhilkinSerg** [comment]
While disassembly time should probably be decreased, 10 seconds seems too little for a handful of nuts.

Disassembly was introduced to de-clutter recipe menu (https://github.com/CleverRaven/Cataclysm-DDA/pull/31034). I am not a fan of bringing clutter back.

You can batch disassemble nuts dropped from map tile.

**Comment by RDru** [comment]
Right, it's "handful". 
I agree with clutter but having to take items, drop them somewhere where there are none and then disassemble them is kinda too many action when it should be simple. I also wanted this to benefit from default crafting benefits like using table.
Anyway, closing this.

**Comment by Rail-Runner** [comment]
> I also wanted this to benefit from default crafting benefits like using table.

At least for this part, more straightforward solution would be allowing crafting places like tables to help with disassembly as well.

**Comment by ZhilkinSerg** [comment]
We can also add mass disassemble menu.

**Comment by sharkfinsouperman** [comment]
@ZhilkinSerg, 0.E has been released and we're still waiting for mass disassembly of nuts. Meanwhile, it takes 200 keystrokes to dissemble 100 peanuts, no tool is needed (we're using our teeth to remove the shells?), and companions don't assist.

There have been dozens of new crafting recipes added since disassembly was changed to ["Allow nut shelling via disassembly to reduce amount of recipe cluttering."](https://github.com/CleverRaven/Cataclysm-DDA/pull/31658), some being very gameplay specific and redundant to the majority of players, making the change redundant and lacking in any benefit in relation to the tediousness it has created.

Reverting to the original crafting state would be nothing but beneficial in reducing tedious actions and improving QOL while having zero negative effects beyond there being a few additional recipes.

**Comment by ZhilkinSerg** [comment]
Yeah, go implement it. I am all in.

**Comment by sharkfinsouperman** [comment]
My method of implementation would simply be reverting the change. Would you be okay with that?

**Comment by ZhilkinSerg** [comment]
That is not implementation of mass disassembly menu.

**Comment by sharkfinsouperman** [comment]
I took a look at  [how mass disassembly of ammo was implemented](https://github.com/CleverRaven/Cataclysm-DDA/commit/30c5e2465b8af1a053b6e6d9dd0b453ad389582c) and, while it would be an easy task for someone with more experience and familiarity with the game code, it's beyond my abilities.

Would submitting an implementation request be appropriate?

**Pull request closed without merging**